scripts/security/check_secret_expiration.py

```
@@ -0,0 +1,584 @@
+    risk_level: RiskLevel
+    last_rotation: datetime
+    next_rotation: datetime
+    rotation_interval_days: int
+    owner: str
+    vault_path: str
+    notes: str = ""
+
+    @property
+    def days_until_expiration(self) -> int:
+        """Days until secret expires."""
+        delta = self.next_rotation - datetime.now()
```

Ruff (0.14.4)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)